[PL-43116]: Doc updates for audit trail and audit stream done as part of doc review (#4275)
harness/developer-hub: docs/platform/governance/audit-trail/audit-streaming.md

- **Edit**: Select a different streaming destination or make changes to the existing destination.

> - **Delete**: Delete the audit stream destination. Streaming destination should be marked inactive before it can be deleted.

Reviewer: ```suggestion
- **Delete**: Delete the audit stream destination. You must set the audit stream destination to inactive before you can delete it.
```
